app/code/Magento/Sales/Test/Unit/Model/Order/ItemTest.php

```diff
@@ -348,4 +348,22 @@ public function getItemQtyVariants()
             ]
         ];
     }
+
+    /**
+     * Test getPrice() method
+     */
+    public function testGetPrice()
+    {
+        $price = 9.99;
+        $this->model->setPrice($price);
+        $this->assertEquals($price, $this->model->getPrice());
```

Comment:
Please move each assertion to a separate test method, with its own description and name.

Reply:
@eduard13 
I have created two separate methods instead of one.